Link_sinFade_Library.ino

```diff
@@ -397,16 +406,46 @@ void ArrayPrintFloat(float Array[]) {  // Prints any array of three elements.
 }
 //////////////////////////////////////////
 int delayKnob(float delayNum ) {                  //**** consider the map()     // map( analogRead(),   0,1023, 10,940)
-  float knobvalue = (analogRead(SLOWNESS) / 110.0) + 0.1;        // analogRead(3) = 0 to 1023.
-  int delayout = delayNum * knobvalue;                           // delayKnob(100) = 10 to 940.
-  return delayout;                                   // 'return' is not a function... It is an statement - no paren needed.
+  static unsigned long last = 0;
+  static int delay_f;
+
+  if ( millis() - last > 100 ) {
+    float knobvalue = (analogRead(SLOWNESS) / 110.0) + 0.1;        // analogRead(3) = 0 to 1023.
+    if (BRIGHTNESS == -1) knobvalue = SLOWNESS_OVERRIDE ; // override, force mid speed
+    delay_f = delayNum * knobvalue;   // delayKnob(100) = 10 to 940.
+    last = millis();
+  }
```

Comment:
In other words, do `analogRead()` only every 100 millis. Did the same for brightness